Toxic Masculinity In Nerd Culture

```diff
@@ -0,0 +1,47 @@
+Toxic Masculinity in Nerd Culture
+
+“Nerd culture is now part of the zeitgeist. Come join us and learn why you got to get over your idea of what a nerd is, dude.” < dude is meant ironically.
+
+Examples to keep in mind throughout discussion:
```

Comment:
Are these examples for you?  I like that you have this list as things that you can refer to and bring up throughout the session.